lib/ignore/ignore.go

```diff
@@ -22,16 +22,29 @@ import (
 	"github.com/syncthing/syncthing/lib/sync"
 )
 
+const (
+	Exclude int32 = 0
```

Comment:
Using the type/consts from above (maybe made exported then). Also not as a bitmask I think, unless `Delete` makes sense in the absence of `Ignore` somehow?